_sources/Unit1-Using-Objects-and-Methods/topic-1-13-constructors.rst

```diff
@@ -241,11 +241,11 @@ Constructors are said to be **overloaded** when there are multiple constructors,
    :answer_d: Turtle t = new Turtle(world1,50,150);
    :answer_e: Turtle t = new Turtle(50,150,world1);
    :correct: e
-   :feedback_a: There is no Turtle constructor that takes no parameters according to the figure above.
-   :feedback_b: There is no Turtle constructor that takes 2 parameters according to the figure above.
+   :feedback_a: There is no Turtle constructor that takes no arguments according to the figure above.
+   :feedback_b: There is no Turtle constructor that takes 2 arguments according to the figure above.
    :feedback_c: This would initialize the Turtle to the middle of the world, not necessarily coordinates (50,150).
-   :feedback_d: Make sure the order of the parameters match the constructor signature above.
-   :feedback_e: This matches the second constructor above with the parameters of x, y, and world.
+   :feedback_d: Make sure the order of the arguments match the constructor signature above.
+   :feedback_e: This matches the second constructor above with the arguments of x, y, and world.
```

Comment:
I think this one actually should be "parameters" since we're talking about the names in the signature.

```suggestion
   :feedback_e: This matches the second constructor above with the parameters of x, y, and world.
```